properties.go

```diff
@@ -94,6 +94,15 @@ const (
 	// multicast on startup.
 	EnvVarMulticastEnabled = "SMUDGE_MULTICAST_ENABLED"
 
+	// EnvVarMulticastAnnounceInterval is the name of the environment variable
+	// that describes whether Smudge will attempt to re-announce its presence
+	// via multicast every X seconds.
+	EnvVarMulticastAnnounceInterval = "SMUDGE_MULTICAST_ANNOUNCE_INTERVAL"
+
+	// DefaultMulticastAnnounceInterval is the default value for whether Smudge
```

Comment:
Comments should mention what time units are being used. The rest of the time values use milliseconds. Perhaps this should even use millis for consistency? I'm torn on that one... I'll leave it up to you.